dev_tools/auto_merge.py

```diff
@@ -534,8 +533,8 @@ def attempt_sync_with_master(pr: PullRequestDetails
             "'Update Branch' for me before trying again.")
 
     raise RuntimeError('Sync with master failed for unknown reason. '
-                       'Code: {}. Content: {}.'.format(response.status_code,
-                                                       response.content))
+                       'Code: {}. Content: {!r}.'.format(
+                           response.status_code, response.content))
```

Comment:
Use a format string?

(Applies throughout.)

Reply:
Acknowledged. Wanted to keep the refactoring minimal.